src/smb_mission_planner/mission_recorder.py

```diff
@@ -105,8 +106,8 @@ def removeWaypoint(self, data):
             rospy.logwarn("The waypoint '" + data.mission_name + "/" + data.waypoint_name + "' does not exist.")
             return RemoveWaypointResponse()
 
-    def basePoseCallback(self, pose_stamped_msg):
-        self.base_pose_msg = pose_stamped_msg
+    def basePoseCallback(self, Odometry_msg):
+        self.base_pose_msg = Odometry_msg
```

Comment:
you could maybe also rename the `self.base_pose_msg` to `self.odometry_msg` or similar...
But then you would also need to adapt it later (#L113)